standalone_tests/artifact_object_reference_test.py

```diff
@@ -664,6 +681,8 @@ def test_image_reference_with_preferred_path():
         test_table_slice_reference_artifact,
         test_image_refs,
         test_point_cloud_refs,
+        test_html_refs,
+        test_video_refs,
```

Comment:
The standalone tests currently only get run / asserted if they're added to the regression.yaml.  Just wanted to make sure you knew about that.

Reply:
For a while I was running these manually, but did end up adding them to regression tests: https://github.com/wandb/wandb-testing/blob/master/regression/main/wandb-git/client/standalone_tests/artifact_references/regression.yaml

Note, we also have a test for the dsviz demo as well: https://github.com/wandb/wandb-testing/blob/master/regression/main/wandb-git/client/standalone_tests/dsviz/regression.yaml